includes/templates/single-web-story.php

```diff
@@ -70,15 +74,15 @@
 
 $meta_images = Media::get_story_meta_images();
 
+$poster_portrait = isset( $meta_images['poster-portrait'] ) ? $meta_images['poster-portrait'] : plugin_dir_url( WEBSTORIES_PLUGIN_FILE ) . 'assets/images/fallback-poster.jpg';
```

Comment:
Why not just `Story_Post_Type::get_publisher_logo()`?

Reply:
logo ≠ poster image

Reply:
```suggestion
$poster_portrait = isset( $meta_images['poster-portrait'] ) ? $meta_images['poster-portrait'] : plugins_url( '/assets/images/fallback-poster.jpg', WEBSTORIES_PLUGIN_FILE ) ;
```

Reply:
Fixed in https://github.com/google/web-stories-wp/pull/140/commits/0be1a0d045ae259de3cfdb6be143c11404f5ccb7